addons/payment_stripe/const.py

```diff
@@ -106,3 +106,12 @@
     'YT': 'FR',  # Mayotte
     'MF': 'FR',  # Saint-Martin
 }
+
+# Stripe-specific mapping of currency codes in ISO 4217 format to the number of decimals.
+# Only currencies for which Stripe does not follow the ISO 4217 norm are listed here.
+# See https://docs.stripe.com/currencies#zero-decimal
+CURRENCY_DECIMALS = {
+    'ISK': 2,
+    'MGA': 0,
+    'UGX': 2,
+}
```

Comment:
There are two cases here you should add both docs because https://docs.stripe.com/currencies#zero-decimal link is wrong for UGX and ISK
```suggestion
CURRENCY_DECIMALS = {
    See https://docs.stripe.com/currencies#special-cases
    'ISK': 2,
    'UGX': 2,
    See https://docs.stripe.com/currencies#zero-decimal
    'MGA': 0,
}
```
(could improve bit :thinking: )